src/vue/pages/VerificationCorporate.vue

```diff
@@ -210,7 +210,17 @@ export default {
   computed: {
     ...mapGetters({
       kvEntryCorporateRoleId: vuexTypes.kvEntryCorporateRoleId,
+      isAccountRoleReset: vuexTypes.isAccountRoleReset,
+      previousAccountRole: vuexTypes.kycPreviousAccountRole,
     }),
+    isFormEditable () {
+      return this.isAccountRoleReset ||
+        this.kycState === REQUEST_STATES_STR.rejected
+    },
+    isFormAutocompleted () {
+      return (this.previousAccountRole === this.kvEntryCorporateRoleId) ||
+        (!this.isAccountRoleReset && this.kycState)
```

Comment:
move to another computed, cause idk what this is about

Reply:
Refactored a bit